tests/build_tests/super_errors/expected/subtype_string.res.expected

```diff
@@ -0,0 +1,16 @@
+
+  [1;31mWe've found a bug for you![0m
+  [36m/.../fixtures/subtype_string.res[0m:[2m7:18-20[0m
+
+  5 [2m│[0m }
+  6 [2m│[0m 
+  [1;31m7[0m [2m│[0m let x = takesStr([1;31mOne[0m)
+  8 [2m│[0m 
+
+  This has type: [1;31mvariant[0m
+  But it's expected to have type: [1;33mstring[0m
+  
+  Possible solutions: 
+  - These types are compatible at runtime. You can use the coercion operator [1;33m:>[0m to convert to the expected type [1;33mstring[0m.
+    If you want to use coercion, rewrite the highlighted code to: [1;33m(One :> string)
```

Comment:
this is great! But I'd simplify the wording a bit, it feels a bit verbose:
```suggestion
  - These types are compatible at runtime. You can use the coercion operator to convert to the expected type with [1;33m(One :> string)
```

Reply:
@tsnobip how would you simplify the wording? Can't see anything along those lines in the diff above.

Reply:
Oh, I see now.